InnerEye/ML/configs/segmentation/ProstatePaper.py

```diff
@@ -11,6 +11,9 @@
 fg_classes = ["external", "femur_r", "femur_l", "rectum", "prostate", "bladder", "seminalvesicles"]
 fg_display_names = ["External", "Femur_R", "Femur_L", "Rectum", "Prostate", "Bladder", "SeminalVesicles"]
 
+# Change this string to the name of your dataset on Azure blob storage.
+AZURE_DATASET_ID = "your_dataset"
```

Comment:
here and above - this somehow contravenes the suggestion of subclassing those models

Reply:
Removed `AZURE_DATASET_ID` from these classes.